docker/lib/dependabot/docker/update_checker.rb

```diff
@@ -105,12 +95,15 @@ def version_up_to_date?
       end
 
       def version_tag_up_to_date?(version)
```

Comment:
`version` --> `current_version`?? to disambiguate from `latest_version` inside of this method...

Reply:
This didn't seem particularly confusing to me, I felt it was clear that `version` was the parameter to the method. So I decided not to touch this parameter name. I did change the ones that changed types to add `_tag` to avoid confusion.